src/services/GooglePlacesService.ts

```diff
@@ -81,7 +85,16 @@ class GooglePlacesService {
         },
       })
       .then((response: AxiosResponse) => {
-        const restaurantResults = response?.data?.result;
+        const restaurantData = response?.data;
+        const restaurantResults = restaurantData?.result;
+
+        // check if placeID is valid
+        if (restaurantData.status && restaurantData.status === 'NOT_FOUND') {
```

Comment:
I just found out that you can use optional chaining to eliminate the first condition!
 
```suggestion
        if (restaurantData?.status === 'NOT_FOUND') {
```